modules/nf-core/trycycler/subsample/meta.yml

```diff
@@ -0,0 +1,62 @@
+name: trycycler_subsample
+description: Subsample a long-read sequencing fastq file for multiple assemblies
+keywords:
+  - subsample
+  - fastq
+  - genomics
+tools:
+  - "trycycler":
+      description: Trycycler is a tool for generating consensus long-read assemblies for bacterial genomes
+      homepage: https://github.com/rrwick/Trycycler
+      documentation: https://github.com/rrwick/Trycycler/wiki
+      doi: 10.1186/s13059-021-02483-z
+      licence: ["GPL v3"]
+
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. `[ id:'sample1', single_end:false ]`
+
+  - reads:
+      type: file
+      description: Long-read FASTQ file, optionally gzip compressed
+
+  - out_dir:
+      type: directory
+      description: Output directory for subsampled read sets
+
+  - count:
+      type: integer
+      description: Number of subsampled read sets to output, default 12
+
+  - genome_size:
+      type: string
+      description: Approximate genome size (e.g. 5.5m), default auto
+
+  - min_read_depth:
+      type: float
+      description: Minimum allowed read depth, default 25.0
```

Comment:
Arguments for the tool passed via `args` don't need detailing here.

Reply:
resolved in [d9cc745](https://github.com/nf-core/modules/pull/5654/commits/d9cc745b6a04740355262bcfa49fe0c8a11824c6)